test/extensions/filters/http/grpc_http1_reverse_bridge/reverse_bridge_test.cc

```diff
@@ -43,6 +43,30 @@ class ReverseBridgeTest : public testing::Test {
   Http::MockStreamEncoderFilterCallbacks encoder_callbacks_;
 };
 
+// Verifies that an incoming request with a missing Content Type is rejected
```

Comment:
This tests that the case where a *response* is missing content-type, not the request